android/src/main/java/ie/gov/tracing/nearby/ProvideDiagnosisKeysWorker.java

```diff
@@ -129,15 +128,27 @@ private void updateLastRun() {
   @NonNull
   @Override
   public ListenableFuture<Result> startWork() {
+      // validate config set before running
+      final String server = SharedPrefs.getString("serverUrl", this.context);
```

Comment:
moved this condition up so on first run, if config wasn't set, it won't show a foreground notification